modules/ROOT/pages/dialog-configuration.adoc

```diff
@@ -45,7 +45,7 @@ tinymce.activeEditor.windowManager.open({
 |title |string |required |The title of the dialog. This will display in the header of the dialog.
 |body |`+panel+` or `+tabpanel+` component |required |The specification for the xref:dialog-components.adoc[body component].
 |buttons |`+FooterButton[]+` |required |An array of xref:dialog-footer-buttons.adoc[footer buttons] to render in the footer of the dialog.
-|size |`+'normal'+`, `+'medium'+` or `+'large'+` |optional |default: `+normal+` - Dialog size options.
+|size |`+'normal'+`, `+'medium'+` or `+'large'+` |optional |default: `+normal+`. `+large+` sets the dialog to a large fixed height. `+medium+` sets the dialog to render with only the height required to fits its components. `+normal+` *[does what?]*. `inline` dialogs do not support `size: 'large'`.
```

Comment:
the size configs currently are: 

[normal](https://github.com/tinymce/tinymce/blob/fac57f5f9c591e6cb387ce639ab28010ae96863a/modules/oxide/src/less/theme/components/dialog/dialog.less#L105)
```
width: 95vw
max-height: 100%;
max-width: 480px;

@media @breakpoint-sm (767px) {
    max-height: calc(100vh - @pad-sm (16px) * 2);
    width: calc(100vw - @pad-md (16px));
}
```

[medium](https://github.com/tinymce/tinymce/blob/fac57f5f9c591e6cb387ce639ab28010ae96863a/modules/oxide/src/less/theme/components/dialog/dialog.less#L389)
```
max-width: 800px;
```

[Large](https://github.com/tinymce/tinymce/blob/fac57f5f9c591e6cb387ce639ab28010ae96863a/modules/oxide/src/less/theme/components/dialog/dialog.less#L375)
```
height: 650px;
max-width: 1200px;
```

Reply:
I also create d a fiddle to contain both normal and inline dialog with all 3 size options
https://fiddle.tiny.cloud/mDiaab

Reply:
The [edit](https://github.com/tinymce/tinymce-docs/commit/f17935b3e66bcde403b4fab215bcdd6bb9a3d285) that removes the placeholder text.